src/tree-internal.component.ts

```diff
@@ -415,7 +415,7 @@ export class TreeInternalComponent implements OnInit, OnChanges, OnDestroy, Afte
         this.tree.checked = false;
         this.checkboxElementRef.nativeElement.indeterminate = true;
         this.nodeDraggableService.removeCheckedNodeById(this.tree.id);
-        this.treeService.fireNodeIndetermined(this.tree);
```

Comment:
As  a side-note, related commits like this and the following two could be squashed into one in the future.